### PR TITLE
Fix broken link in docs/howto/mock-services.md

### PR DESCRIPTION
**Pull request by axtsnee**
Changed link target from nonexistent .md file to the file that seems most likely to be the intended target.

### CODE REVIEW

File: docs/howto/mock-services.md [hunk 1]
```diff
@@ -474,7 +474,7 @@ You can find more examples in the `examples` and `test-tests` subproject:
 - [ComposedEmptyMockSpec][link-gh-composed-empty-mock-spec]
 - [PolyMockSpec][link-gh-poly-mock-spec]
 
-[doc-use-modules-and-layers]: use-modules-and-layers.md
+[doc-use-modules-and-layers]: ../datatypes/contextual/index.md
```
Comment (khajavi) [inline docs/howto/mock-services.md:477]: We also need to change the title of the page we are referring to that. The following sentence doesn't reflect that:

> We'll be assuming you've read How to use modules and layers guide.

Comment (axtsnee) [reply docs/howto/mock-services.md:477]: That makes sense. Does this sound better?
"We'll be assuming you read about modules and layers in the contextual types guide."

Comment (khajavi) [reply docs/howto/mock-services.md:477]: Yes, it sounds better.

**Review by khajavi — APPROVED** [review]

File: docs/howto/mock-services.md (hunk 1 above)
Comment (axtsnee) [reply docs/howto/mock-services.md]: Done.

**Review by adamgfraser — APPROVED** [review]

**Pull request merged**